.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
One of two major changes in this PR. The rest of the changes are results of running `lint --fix`